prov/gni/test/rdm_sr.c

```diff
@@ -894,38 +890,41 @@ void do_recvv(int len)
 Test(rdm_sr, recvv)
 {
 	rdm_sr_xfer_for_each_size(do_recvv, 1, BUF_SZ);
+
 }
 
 Test(rdm_sr, recvv_retrans)
 {
 	rdm_sr_err_inject_enable();
 	rdm_sr_xfer_for_each_size(do_recvv, 1, BUF_SZ);
+
 }
 
 /*
-ssize_t (*recvmsg)(struct fid_ep *ep, const struct fi_msg *msg,
-		uint64_t flags);
- */
+  ssize_t (*recvmsg)(struct fid_ep *ep, const struct fi_msg *msg,
+  uint64_t flags);
+*/
```

Comment:
I don't understand why these prototype comments were edited.  Not a big deal but it makes the diff larger and changes from more standard prototype formatting.